src/app/ui/editor/standby_state.cpp

```diff
@@ -429,6 +434,14 @@ bool StandbyState::onSetCursor(Editor* editor, const gfx::Point& mouseScreenPos)
   if (m_decorator->onSetCursor(ink, editor, mouseScreenPos))
     return true;
 
+  // Forbidden
+  Layer* layer = editor->getSite().layer();
+  if (layer && !layer->isEditableHierarchy()) {
+    editor->showMouseCursor(
+      kCustomCursor, skin::SkinTheme::instance()->cursors.forbidden());
+    return true;
+  }
+
```

Comment:
I'm not sure about this, maybe too restrictive here, because even with the forbidden cursor we can still select pixels.